docs/technical-documentation/migration-overview.md

```diff
@@ -0,0 +1,151 @@
+## Islandora 8 Migration Overview
+
+This video provides an overview of the various options available to migrate data into an Islandora 8 installation.
+
+[![Islandora Online: Islandora Migration Tools](https://img.youtube.com/vi/95Bnix-z1zY/0.jpg)](https://www.youtube.com/watch?v=95Bnix-z1zY)
+
+The three options are:
+- REST API
+- Migrate API
+- Islandora Workbench
+
+### REST API
+
+Why use the rest API?
+- Works anywhere
+  - You don’t have to work on the Drupal server. Migrate from your laptop!
+- No PHP required
+  - Use any language that can make an http request. Even cURL will do just fine.
+- JSON
+  - Why use XML if you don’t have to?
+
+BONUS: It’s just Drupal’s REST API
+
+#### Islandora only provides two additional API endpoints
+
+- /media/{mid}/source
+  - PUT a file to this endpoint to create/update a Media’s file
+- /node/{nid}/media/{media_type}/{taxonomy_term}
+  - PUT a file to this endpoint to create/update a Media for a Node
+
+Just be aware, you are writing everything yourself! (In other words you are making all of the migration decisions yourself.)
+
+### Migrate API
+
+Uses the Drupal 8 [Migrate API](https://www.drupal.org/docs/8/api/migrate-api/migrate-api-overview), which "provides services for migrating data from a source system to Drupal 8.
```

Comment:
This builds fine and displays nicely. Just wondering about the start of this sentence. It feels like it is missing something?

Reply:
It reads consistent with the rest of the document. Improvements can always happen later.